stream_alert_cli/logger.py

```diff
@@ -29,13 +30,17 @@ class SuppressNoise(logging.Filter):
     """
 
     def filter(self, record):
-        suppress_starts_with = (
-            'Starting download from S3',
-            'Completed download in'
+        suppressed_messages = (
+            'Starting download from S3*',
+            'Completed download in*',
+            '*triggered an alert on log type*',
+            'Successfully sent * messages to Firehose*'
```

Comment:
Won't we need this for Kinesis streams as well?

Reply:
Aka we might need this to be `Successfully sent * messages to *`

Reply:
@mime-frame nope, Lambda only sends to Firehose

Reply:
@jacknagz could answer that - I don't know of any log message for Kinesis right now?